bot/exts/evergreen/emoji.py

```diff
@@ -1,31 +1,33 @@
-import datetime
 import logging
 import random
+import textwrap
 from collections import defaultdict
-from typing import List, Tuple
+from datetime import datetime
+from typing import List, Optional, Tuple
 
-import discord
+from discord import Color, Embed, Emoji
 from discord.ext import commands
 
 from bot.constants import Colours, ERROR_REPLIES
 from bot.utils.pagination import LinePaginator
+from bot.utils.time import time_since
 
 log = logging.getLogger(__name__)
 
 
-class EmojiCount(commands.Cog):
-    """Command that give random emoji based on category."""
+class Emoji(commands.Cog):
```

Comment:
Is overwriting the import `discord.Emoji`.

Reply:
Good catch!